include/astshim/Object.h

```diff
@@ -63,6 +63,23 @@ class Object {
     Object &operator=(Object &&) = default;
 
     /**
+    Return True if this and `rhs` are the equal
+
+    For two objects be equal, they both must have the same attributes and all contained objects
+    must be equal. However, this test ignores the difference between attributes that are not set
+    and attributes that are set to their default value, because that difference is irrelevant
+    to most behavior.
```

Comment:
You raise a good point. The AST manual is somewhat vague on this and I'm going to play it safe and *not* ignore the difference. That will require a tweak in afw's Transform constructor, but it's probably worth the tweak.